gnovm/pkg/doc/dirs.go

```diff
@@ -91,6 +96,32 @@
 	return dirs
 }
 
+func packageImportsRecursive(root string) []string {
+	res, err := gnoimports.PackageImports(root)
+	_ = err
+
+	entries, err := os.ReadDir(root)
```

Comment:
since we sort, it doesn't matter

Reply:
Why don't we do error handling here?

`gnoimports.PackageImports` does the same `ReadDir`, but we swallow errors from both calls?

Reply:
The original code has a very specific behavior that I tried to preserve this way.
Since refactoring this with a `fs.WalkDir` will be tricky and we will replace this with the packages loader, can I keep as-is?

Reply:
Was the original also swallowing errors?

Reply:
Yes, because the original was reading the `require` directives and `gno mod tidy` which populate them, prints a multierr, thus "ignoring" malformed imports in gno files

if I error out, I have to change the tests behavior

Reply:
I improved this error handling in 9e62b83edbd15535c6409ef488f16cc8f70fa0f3